tests/test_redirects.py

```diff
@@ -0,0 +1,74 @@
+def setup_app(app):
+    from lona import LonaView
+
+    # root
+    @app.route('/')
+    class RootView(LonaView):
+        def handle_request(self, request):
+            return 'ROOT'
+
+    @app.route('/redirect-to-root/')
+    class RedirectToRoot(LonaView):
+        def handle_request(self, request):
+            return {
+                'redirect': '/',
+            }
+
+    # absolute url
+    @app.route('/foo/')
+    class FooView(LonaView):
+        def handle_request(self, request):
+            return 'FOO'
+
+    @app.route('/redirect-to-foo/')
+    class RedirectToFooView(LonaView):
+        def handle_request(self, request):
+            return {
+                'redirect': '/foo/',
+            }
+
+    # relative url
+    relative_redirected = False
+
+    @app.route('/bar/')
+    class BarFooView(LonaView):
+        def handle_request(self, request):
+            nonlocal relative_redirected
+
+            if not relative_redirected:
+                relative_redirected = True
+
+                return {
+                    'redirect': '.',
```

Comment:
Let's add one more test with relative redirection `'redirect': './baz'`.

Also I have a question should `'redirect': './baz'` behave the same as  `'redirect': 'baz'`?

Reply:
Ack!
Yes i think so.

Reply:
Asking because now it's not.

Reply:
I saw it coming :D thanks for reporting, i'll fix it

Reply:
Fixed! And i added a test for this use case